.swiftlint.yml

```diff
@@ -0,0 +1,69 @@
+whitelist_rules:
+  - closing_brace
+  - control_statement
+  - custom_rules
+  - cyclomatic_complexity
+  - dynamic_inline
+  - empty_count
+  - empty_parameters
+  - file_length
+  - force_cast
+  - force_try
+  - force_unwrapping
+  - function_body_length
+  - function_parameter_count
+  - leading_whitespace
+  - legacy_cggeometry_functions
+  - legacy_constant
+  - legacy_constructor
+  - legacy_nsgeometry_functions
+  - line_length
+  - mark
+  - nesting
+  - operator_whitespace
+  - private_unit_test
+  - return_arrow_whitespace
+  - statement_position
+  - trailing_newline
+  - trailing_semicolon
+  - type_body_length
+  - type_name
+  - unused_enumerated
+  - valid_ibinspectable
+  - colon
+  - closure_spacing
+  - comma
+  - empty_parentheses_with_trailing_closure
+  - implicit_return
+  - opening_brace
+  - operator_usage_whitespace
+  - redundant_discardable_let
+  - redundant_nil_coalescing
+  - redundant_optional_initialization
+  - redundant_void_return
+  - trailing_comma
+  - unused_closure_parameter
+  - void_return
+  - identifier_name
+
+excluded: # paths to ignore during linting. overridden by `included`.
+  - Pods
+  - .git
+  - ReactiveArchitectureTests
```

Comment:
on the components side, we allow swiftlint to lint our test code, but with custom rule sets.
Test code is still code that needs to follow standards

Reply:
you're strict lol. Good to know though. I 'll leave for now.